src/libexpr/primops/fetchTree.cc

```diff
@@ -431,6 +439,18 @@ static RegisterPrimOp primop_fetchTree({
     .experimentalFeature = Xp::FetchTree,
 });
 
+void prim_fetchFinalTree(EvalState & state, const PosIdx pos, Value * * args, Value & v)
+{
+    fetchTree(state, pos, args, v, {.isFinal = true});
+}
+
+static RegisterPrimOp primop_fetchFinalTree({
+    .name = "fetchFinalTree",
+    .args = {"input"},
+    .fun = prim_fetchFinalTree,
+    .internal = true,
+});
+
```

Comment:
```suggestion
```

Couldn't we construct the primop on the fly when loading `call-flake.nix`?
That way we don't need `internal` and `internalPrimOps`.

Reply:
That seemed tricky since this happens in `libfetcher` rather than `libexpr`, so it's not clear how to construct a singleton value. Or we could allocate a new PrimOp every time, but that would leak.